tools/ci_build/github/linux/docker/Dockerfile.manylinux_2_27_cpu

```diff
@@ -0,0 +1,13 @@
+FROM onnxruntimebuildcache.azurecr.io/manylinux_2_27_x86_64:latest
```

Comment:
"onnxruntimebuildcache.azurecr.io/manylinux_2_27_x86_64:latest" What is this? Did you manually generate the base image?

Reply:
copy the changes to the build scripts